### PR TITLE
Integration Test that sends QRL between node_1 and node_2

### PR DESCRIPTION
**Pull request by randomshinichi**
Interestingly enough, the sending address gets debited 9 QRL, not 10. I'm not sure if this has to do with the fee calculation (sent with amount=10  fee=1).

Further improvements:
ask other nodes what are the new balances of A and B
Travis doesn't run this integration test automatically.

### CODE REVIEW

File: qrl_testing/NodeInterface.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    """
+    def __init__(self, ip, port=9009, debug=False):
+       """
+       server format: ip:addr
+       """
+       self.debug = debug
+       self.channel = grpc.insecure_channel(":".join([ip, str(port)]))  # 127.0.0.1, 9009 -> "127.0.0.1:9009"
+       self.stub = qrl_pb2_grpc.PublicAPIStub(self.channel)
+
+    def send(self, from_addr:AddressBundle, to_addr:bytes, amount, fee):
```
Comment (jleni) [inline qrl_testing/NodeInterface.py]: fee is not used in the code and later the fee is hardcoded to 1

File: qrl_testing/NodeInterface.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        transferCoinsReq = qrl_pb2.TransferCoinsReq(
+            address_from=from_addr.address,
+            address_to=to_addr,
+            amount=amount,
+            fee=1,
```
Comment (jleni) [inline qrl_testing/NodeInterface.py]: here is hardcoded.. that could be a problem because if the intention is to use this class in other tests... this could be problematic

File: qrl_testing/grpc_compile.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import grpc
```
Comment (jleni) [inline qrl_testing/grpc_compile.py]: It does not seem related in any way to the PR or test, is it?

File: manual_grpc.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
+import grpc
```
Comment (jleni) [inline manual_grpc.py]: If it is really necessary to include this file in the PR, let's put it in some directory to keep things ordered, otherwise just delete.

File: manual_nodeinterface.py [hunk 5]
```diff
@@ -0,0 +1,31 @@
+import grpc
```
Comment (jleni) [inline manual_nodeinterface.py]: If this is about a test for the node interface, I would prefer an automaticunit test. Still, I think it should go in some directory

File: qrl_testing/IntegrationTest.py [hunk 6]
```diff
@@ -25,20 +25,75 @@
...
+
+        wallet_address_file = os.path.join(volumes_path, "wallet_address")
+        with open(wallet_address_file) as f:
+            self.Qaddress = f.readline().strip()
+        
+        self.wallet_dir = os.path.join(volumes_path, "wallet/")
+
+    def update(self, log_entry: LogEntry):
+        if log_entry.synced == 'synced':
+            self.synced = True
```
Comment (jleni) [inline qrl_testing/IntegrationTest.py]: This whole change implies that the only state that is important is synced... But this is a base class for all tests and we do care about all other states. The idea of cleaning up and avoiding strings is good but let's make them constants.. Let's not hide the fact that we have many possibilities.

File: qrl_testing/grpc_compile.py (hunk 3 above)
Comment (randomshinichi) [reply qrl_testing/grpc_compile.py]: We will need this very much for Travis. the docker images clone the QRL main repo, but you don't want the travis test runner to clone the QRL main repo too just for qrl_pb2 and qrl_pb2_grpc, right?

File: qrl_testing/NodeInterface.py (hunk 1 above)
Comment (randomshinichi) [reply qrl_testing/NodeInterface.py]: fixed
